.github/workflows/Docker.yaml

```
@@ -0,0 +1,80 @@
+name: Docker
+on:
+  pull_request:
+    paths:
+      - "**"
+      - "!docs/**"
+      - "!README.md"
+      - "!CONTRIBUTING.md"
```

```suggestion
      - "!CONTRIBUTING.md"
      - "!.github/**"
```
Without this line, there is no point in explicitly specifying: ".github/workflows/Docker.yaml"
See also [previous workflow](https://github.com/tier4/scenario_simulator_v2/pull/1317/files#diff-c3f589b0287668535695b316897f85ac93aaf209dd7ccabc318f9c9d6846b52dL13-L14).